middle_end/flambda2/simplify/simplify_set_of_closures.ml

```diff
@@ -134,10 +135,9 @@ end = struct
       (* When not lifting (i.e. the bound names are variables), we need to
          create a fresh set of irrelevant variables, since the let-bound names
          are not in scope for the closure definition(s). *)
-      List.map
-        (fun closure_bound_names ->
-          Closure_id.Map.map_sharing Bound_name.rename closure_bound_names)
-        closure_bound_names_all_sets
+      (* List.map (fun closure_bound_names -> Closure_id.Map.map_sharing
```

Comment:
Is this commented out intentionally?

Reply:
Ah whoops no, that was from seeing if everything works ok without this second set of irrelevant closure variables.  I will remove this.